packages-native/safe-area-view/src/SafeAreaView.tsx

```diff
@@ -10,10 +10,8 @@ export const SafeAreaView = (props: SafeAreaViewProps<SafeAreaViewStyle>): JSX.E
     const styles = flattenStyles(defaultSafeAreaViewStyle, customStyles);
 
     return (
-        <Fragment>
-            <ReactSaveAreaView style={{ flex: 0, backgroundColor: styles.unsafeAreaTop.backgroundColor }} />
-            <ReactSaveAreaView style={{ ...styles.container, flex: 1 }}>{props.content}</ReactSaveAreaView>
-            <ReactSaveAreaView style={{ flex: 0, backgroundColor: styles.unsafeAreaBottom.backgroundColor }} />
-        </Fragment>
+        <ReactSaveAreaView style={{ flex: 1 }}>
```

Comment:
Does it work in all the conditions?

Reply:
It does the job for the use cases discussed with Danny and Wesley. The use case of having a different background color for the top and bottom unsafe area (meaning you would want to change the flex 1) is not important according to Danny. The question is actually what our definition of the container is for this widget: the whole widget (safe + unsafe areas) or just the safe area. Wesley and I came to the conclusion to go for the last one.